streaming/text/convert/enwiki.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2022 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""English Wikipedia 2020-01-01 streaming dataset conversion script."""
+
+import os
+from argparse import ArgumentParser, Namespace
+from typing import List
+
+from tqdm import tqdm
+
+from streaming.base import MDSWriter
+from streaming.base.util import get_list_arg
+
+
+def parse_args() -> Namespace:
+    """Parse command-line arguments.
+
+    Args:
+        Namespace: command-line arguments.
+    """
+    args = ArgumentParser()
+    args.add_argument(
+        '--in_root',
+        type=str,
+        required=True,
+        help='Directory path to store the input dataset',
+    )
+    args.add_argument(
+        '--out_root',
+        type=str,
+        required=True,
+        help='Directory path to store the output dataset',
+    )
+    args.add_argument(
+        '--compression',
+        type=str,
+        default='zstd:7',
+        help='Compression algorithm to use. Default: zstd:7',
+    )
+    args.add_argument(
+        '--hashes',
+        type=str,
+        default='sha1,xxh64',
+        help='Hashing algorithms to apply to shard files. Default: sha1,xxh64',
+    )
+    args.add_argument(
+        '--size_limit',
+        type=int,
+        default=1 << 25,
+        help='Shard size limit, after which point to start a new shard. Default: 1 << 27',
```

Comment:
nit: `1 << 25` in `help` message

Reply:
Done

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/23/commits/926d9779baed22f1e9392a6b96b43cfc3f63d68e.